less-builder.js

```diff
@@ -20,33 +36,63 @@ var cssInject = [
 ].join(';');
 
 exports.bundle = function (loads, compileOpts, outputOpts) {
-	var _this = this;
+    var loader = this;
```

Comment:
I agree with the usage of "loader" instead of "_this" (I just used `_this` because eslint kept complaining) but said change is generic, doesn't belong mandatorily to this PR. I'd rather have PRs that change only what they really need to change.

Please revert this and, if you wish, submit a new PR just to replace the variable name.